app/services/RuleProvisionerService.scala

```diff
@@ -32,21 +32,18 @@ class RuleProvisionerService(
       case (category, rules) => {
```

Comment:
`category` is no longer used here. Do you still need to do the `groupBy` on the line above or can it be simplified?

Reply:
We can make category a no-op, but because we produce a single regex matcher per category, we still need the groupBy.